### PR TITLE
Created first question schema and route

### PR DESCRIPTION
**Pull request by a-oboh**

### CODE REVIEW

**Comment by timolinn** [comment]
@a-oboh You were supposed to work from the dev branch or created a branch.

File: src/routes/firstQuestions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const firstQuestionRouter = express.Router;
+const registerQuestions = require('../components/questions/firstQuestionModel');
+
+//post questions on register
+firstQuestionRouter.post('/register-question', (req, res) => {
```
Comment (timolinn) [inline src/routes/firstQuestions.js]: Are we going to have a secondQuestionRouter? 

I think it's a good idea to have only one questions model/controller/router that takes care of all the questions.

Same goes for answers.

**Comment by a-oboh** [comment]
Okay, I'll change it to one file for all 
 
  On Wed, Apr 10, 2019 at 1:31 PM, Timothy Onyiuke<notifications@github.com> wrote:   
@timolinn commented on this pull request.

@a-oboh kindly look at my comments

In src/routes/firstQuestions.js:
> @@ -0,0 +1,14 @@
+const express = require('express');
+const firstQuestionRouter = express.Router;
+const registerQuestions = require('../components/questions/firstQuestionModel');
+
+//post questions on register
+firstQuestionRouter.post('/register-question', (req, res) => {

Are we going to have a secondQuestionRouter?

I think it's a good idea to have only one questions model/controller/router that takes care of all the questions.

Same goes for answers.

**Comment by a-oboh** [comment]
Yes I'm going to switch to the dev branch and move everything there. Also we need to do some planning on slack

**Pull request closed without merging**